testing/trino-faulttolerant-tests/src/test/java/io/trino/faulttolerant/sqlserver/BaseSqlServerFailureRecoveryTest.java

```diff
@@ -34,6 +40,21 @@ public BaseSqlServerFailureRecoveryTest(RetryPolicy retryPolicy)
         super(retryPolicy);
     }
 
+    @Override
+    @Test(invocationCount = INVOCATION_COUNT)
+    public void testUserFailure()
+    {
+        // because we have arithmetic push down, now we will get exception not on trino side, but from sql server,
+        // so error message will be different
+        assertThatThrownBy(() -> getQueryRunner().execute("SELECT * FROM nation WHERE regionKey / nationKey - 1 = 0"))
```

Comment:
pass a session in a base test to disable the predicate pushdown. Then you don't need to override the test.

Reply:
done